presentation/backend/src/test/PresentationManager.test.ts

```diff
@@ -105,6 +105,21 @@ describe("PresentationManager", () => {
         const cacheConfig = {
           mode: HierarchyCacheMode.Memory,
         };
+        const formatProps = {
+          composite: {
+            includeZero:true,
+            spacer:" ",
+            units: [{label:"'",name:"IN"}],
+          },
+          formatTraits:"KeepSingleZero|KeepDecimalPoint|ShowUnitLabel",
+          precision:4,
+          type:"Decimal",
+          uomSeparator:"",
+        };
+        const defaultFormats = {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+          LENGTH: { unitSystems: [PresentationUnitSystem.BritishImperial], format: formatProps },

```

Comment:
for the matter of these tests, I think we could pass `length` instead of `LENGTH` to avoid the need of "eslint-disable"